### PR TITLE
add reporting location

### LINKED ISSUE #16: Handle 'report an issue'
Get '/api/issue' and send its payload to 'admins email'.

To do it:
- add ADMIN_EMAIL to configuration
- setup email sending
- parse and send payload

### PR DESCRIPTION
**Pull request by kacper-cyra**
Closes #16

### CODE REVIEW

File: tests/e2e_tests/e2e_test_config.yml [hunk 1]
```diff
@@ -14,7 +14,17 @@ LANGUAGES:
...
 ROUTE_OVERWRITES:
   /static/map.js: https://cdn.jsdelivr.net/npm/@problematy/goodmap@0.1.6
+
+SMTP:
+  PORT: 465
+  SERVER: "smtp.gmail.com"
+  ADDRESS: "my@gmail.com"
+  PASSWORD: "PA$$WORD"
+  DEFAULT_SENDER: "sender@gmail.com"
+  ADMIN_EMAIL: "help.desk@gmail.com"
```
Comment (raven-wing) [inline tests/e2e_tests/e2e_test_config.yml:27]: After adding `PLUGINS`, this should no longer be necessary.

Comment (kacper-cyra) [reply tests/e2e_tests/e2e_test_config.yml:27]: Yup, I thought about this, but later I got reflection, that we are testing if app starts at all. So should we test if only base part of an app works, or should we test whole app. If we test only a part of it, then how we are sure if someone who will want to use reporting location won't end up with broken app

Comment (raven-wing) [reply tests/e2e_tests/e2e_test_config.yml:27]: Oh. I totally agree that we should test whole app. :)
Yet - there is no e2e test yet (which should be here).

Comment (raven-wing) [reply tests/e2e_tests/e2e_test_config.yml:27]: Created #30 to track that.

And to be honest - I don't have good idea how to do this. (I have ideas, but would not say they are good). So please enjoy taking part in #30 discussion.

Comment (raven-wing) [reply tests/e2e_tests/e2e_test_config.yml:27]: And summing it up. I'd remove this config and plugin (which are not tested now) and move it to this second issue.
Which should be taken right away.

Comment (kacper-cyra) [reply tests/e2e_tests/e2e_test_config.yml:27]: Done

Comment (raven-wing) [reply tests/e2e_tests/e2e_test_config.yml:27]: It seems that it's still here. Did you forget do `commit push`?

File: goodmap/core_api.py [hunk 2]
```diff
@@ -15,12 +15,13 @@ def core_pages(database, languages, email_service):
     core_api_blueprint = Blueprint('api', __name__, url_prefix="/api")
     core_api = Api(core_api_blueprint, doc='/doc', version='0.1')
 
-    @core_api.route("/locations:report", methods=["POST"])
-    @expects_json(report_location_schema)
-    def report_location():
-        request_body = request.get_json()
-        email_service.send_report_location_email(request_body)
-        return "", 200
+    if email_service:
```
Comment (raven-wing) [inline goodmap/core_api.py:18]: I see a big misunderstanding here. What I meant was only to remove it from test config.
As this is an integral part of code.

File: goodmap/goodmap.py [hunk 3]
```diff
@@ -14,7 +14,7 @@ def create_app(config_path):
 
     app = platzky.create_app_from_config(config)
     extend_app_db(app)
-    app.email_service = EmailService(app.config["SMTP"], app.sendmail)
+    app.email_service = EmailService(app.config["SMTP"], app.sendmail) if 'sendmail' in app.config["PLUGINS"] else None
```
Comment (raven-wing) [inline goodmap/goodmap.py:17]: Same here. Need to discuss it further.

**Comment by raven-wing** [comment]
Issue was closed with another PR

**Pull request closed without merging**